### ISSUE TITLE
May I see the source code?

### ISSUE THREAD

**Issue by LEOcab** [Dudenn/TreeGravity#9]
Hello kind sir. I would like the source code of TreeGravity so that I may make it compatible with the custom trees produced by various world generator plugins. The players on my server would greatly appreciate this feature. However, your GitHub page seems empty other than a readme.md file. If you share your code, I will share whatever modifications I make with you.

Thank you!

**Comment by TheNewJavaman** [comment]
I agree. If you shared the source code, I would be able to update this plugin to 1.16 and submit a pull request.

**Comment by terencekam** [comment]
agree
